scripts/model_gen_ng/model_gen_ng/clangfmt.py

```diff
@@ -0,0 +1,12 @@
+import subprocess
+
+
+def reformat(source: str) -> str:
+    """Applies clang-format to the given source"""
+    return subprocess.check_output(
+        [
+            "clang-format",
+        ],
+        input=source,
+        encoding="utf-8",
```

Comment:
* [ ] Let's make this soft-fail if `clang-format` is not available.
* [ ] Is this worth a separate file?

Reply:
Moved this to `main.py` and implemented soft-fail